charts/dependency-track/Chart.yaml

```diff
@@ -14,3 +14,7 @@ maintainers:
 - name: nscuro
   email: nscuro@protonmail.com
   url: https://github.com/nscuro
+dependencies:
+- name: common
+  repository: oci://registry-1.docker.io/bitnamicharts
+  version: ^2.19.1
```

Comment:
Is there a way to achieve the desired outcome without adding 3rd party dependencies?

Reply:
Yes, but in my opinion, that's unfortunate because Bitnami is quite stable regarding breaking changes. If it's pegged to major version 2, you can benefit from the fixes and improvements of that library. During the build, the entire library is downloaded, unpacked, and added to the final Helm chart, so it serves merely as a build dependency. The resulting chart is hermetically self-sufficient, eliminating any risk of issues related to unavailability, etc.

However, if you want to maintain control, you can add it as a subchart—the result will be the same, but this approach offers greater control over the origin, and, if necessary, it allows for manual updates.